src/chatdbg/util/prompts.py

```diff
@@ -2,9 +2,10 @@
 import os
 from chatdbg.util.config import chatdbg_config
 from .text import truncate_proportionally
+from types import NoneType
+from typing import Any, Callable, List, Union
 
-
-def _wrap_it(before, x, after="", maxlen=2048):
+def _wrap_it(before: str, x: Union[str, NoneType], after: str="", maxlen: int=2048) -> str:
```

Comment:
Simplify `Union[T, NoneType]` into `Optional[T]`? Can't `None` be used directly instead of `NoneType`?

Reply:
Yes and yes